internal/commands/permission.go

```diff
@@ -203,6 +213,10 @@ func checkCmdFunc(cmd *cobra.Command, args []string) error {
 		ctx = requestmeta.AddRequestHeaders(ctx, requestmeta.RequestDebugInformation)
 	}
 
+	if requestID != "" {
+		ctx = requestmeta.WithRequestID(ctx, requestID)
+	}
```

Comment:
to avoid the repetition, could we do this earlier in the stack? perhaps using cobras' lifecycle hooks?

Reply:
Done